### DISCUSSION TITLE
Многократное нажатие клавиш

### DISCUSSION THREAD

**Discussion by IamUnloaded** [OGSR/OGSR-Engine#428] [Q&A]
Здравствуйте.
Имеется-ли реализация многократного нажатия клавиши для выполнения действия? Ну чтоб понятнее - дрочить клавишу, с накоплением индикатора заполнения.

**Comment by joye-ramone** [comment]
Нет. в движке такого нету. Но в скриптах легко делается

**Comment by IamUnloaded** [comment]
**joye-ramone**
Не знаком, пока, со всеми возможностями OGSR, подскажите, где посмотреть описания\примерчики по новым возможностям обработки ввода?
